### ISSUE TITLE
Humidification model

### ISSUE THREAD

**Issue by martin-mosteiro** [architecture-building-systems/CityEnergyAnalyst#1267]
There is currently no humidification model in CEA, just a simple constant estimate of 15 W/(kg/h). Under some specific conditions (buildings with centralized air-based heating), this has caused issues (extremely high auxiliary loads).

A humidification model would avoid these issues.

**Comment by gabriel-happle** [comment]
@JIMENOFONSECA No, currently for `humidification` we are still using some line of code from legacy as mentioned by @martin-mosteiro.
Irrespective of the actual amount of water needed, there is an electric power consumption per kg/h ventilation air flow.
As humidification has a different mechanism as dehumidification we would need a component model for that.
In Singapore this is not an issue, as we never have humidification. But in Zurich this has become an issue in some cases, because we now have the moisture balance.

For @martin-mosteiro 's case we could lower the bounds of comfort (they are also from legacy 30-70% relative humidity) or we could consider the enthalpy of evaporation and maybe some efficiency to translate into electric energy for evaporation.

**Comment by martin-mosteiro** [comment]
@JIMENOFONSECA Yep, what @gabriel-happle said, basically. This only seems to have a big effect on buildings with air-based heating, which is not present in any of our case studies at the moment, so I'm giving it a low priority. But I'm keeping the issue open in case it does appear in other cases.

**Comment by JIMENOFONSECA** [comment]
Got it, thanks!

Kind regards,

Dr. Jimeno A. Fonseca
Senior Researcher and Project Coordinator
Multi-Scale Energy Systems for Low-Carbon Cities (MuSES)
Future Cities Laboratory
Singapore ETH-Centre

Chair of Architecture and Building Systems
Institute of Technology in Architecture
ETH Zurich

Scientific Advisory Committee
Energy and Sustainability 2017

On 31 May 2018, 15:52 +0800, Martín Mosteiro-Romero <notifications@github.com>, wrote:

@JIMENOFONSECA<https://github.com/JIMENOFONSECA> Yep, what @gabriel-happle<https://github.com/gabriel-happle> said, basically. This only seems to have a big effect on buildings with air-based heating, which is not present in any of our case studies at the moment, so I'm giving it a low priority. But I'm keeping the issue open in case it does appear in other cases.
